git-ftp

```diff
@@ -492,7 +492,12 @@ upload_local_sha1() {
 }
 
 pre_push_hook() {
-	local hook='.git/hooks/pre-ftp-push'
+	local hooks_dir="$(git config core.hooksPath)/"
```

Comment:
The additional `/` at the end causes `hooks_dir` to never be empty. So the default setting to `.git/hooks` doesn't work.

Reply:
Hello

Thanks for you review. I've patched and remove trailing /